src/components/PageWizard/PageWizardStep/PageWizardStep.jsx

```diff
@@ -132,6 +137,12 @@ const PageWizardStep = ({
           {i18n.submit}
         </Button>
       )}
+      {afterFooterContent && (
+        <div
+          className={`${iotPrefix}--page-wizard--content--after-footer--content`}>
+          {afterFooterContent}
+        </div>
+      )}
```

Comment:
> Looks good, just one minor thing it RTL mode where spacing is missing between the icon and the text:
> 
> ![image](https://user-images.githubusercontent.com/5167091/96734120-7c964600-13ba-11eb-8f26-f48eb40f5cad.png)

@bjornalm I believe that's just part of the story, the rendered node is fully customizable here